### ISSUE TITLE
Not seeing Attach option in  Lens: 5.5.3-latest.20220602.2

### ISSUE THREAD

**Issue by santikris2003** [kontena/lens#5586]
Not seeing Attach  option in  Lens: 5.5.3-latest.20220602.2 as shown in screenshot to attach POD for bash shell access to the POD 
<img width="268" alt="Screen Shot 2022-06-09 at 12 44 29 PM" src="https://user-images.githubusercontent.com/51011247/172790767-d14f318b-9268-4731-b844-07f889aedf8c.png">

**Comment by Nokel81** [comment]
What platform are you on? Windows? Can you reinstall the application?

**Comment by santikris2003** [comment]
Mac OS

**Comment by Nokel81** [comment]
If you fully quit lens and run it from the command line can you post some logs?

**Comment by santikris2003** [comment]
If you can let me know what to run from command line will run and post the
logs.

**Comment by Nokel81** [comment]
`DEBUG=true /Applications/Lens.app/Contents/MacOS/Lens`

**Comment by santikris2003** [comment]
DEBUG=true /Applications/Lens.app/Contents/MacOS/Lens
info:    ▪ 📟 Setting Lens as protocol client for lens:// +0ms
info:    ▪ 📟 Protocol client register succeeded ✅ +3ms
debug:   ▪ [APP-MAIN] configuring packages +0ms
info:    ▪ [EXTENSIONS-STORE]: LOADING from  ... +8ms
info:    ▪ [EXTENSIONS-STORE]: LOADED from /Users/sankris2_1/Library/Application Support/Lens/lens-extensions.json +4ms
info:    ▪ [FILESYSTEM-PROVISIONER-STORE]: LOADING from  ... +4ms
info:    ▪ [FILESYSTEM-PROVISIONER-STORE]: LOADED from /Users/sankris2_1/Library/Application Support/Lens/lens-filesystem-provisioner-store.json +2ms
debug:   ▪ [APP-MAIN] initializing ipc main handlers +2ms
debug:   ▪ [APP-MAIN] Lens protocol routing main +0ms
info:    ▪ 🚀 Starting Lens from "/Applications/Lens.app/Contents/MacOS/Lens" +47ms
info:    ▪ 🐚 Syncing shell environment +0ms
SantoshK$

**Comment by Nokel81** [comment]
From those logs it looks like you did not fully quit lens before running that command. Please quit lens from the tray icon.

**Comment by santikris2003** [comment]
yes you are correct , now quitted the Lens app from tray icon and run this command DEBUG=true /Applications/Lens.app/Contents/MacOS/Lens getting this Lens Error 

Could not load extensions: npm WARN checkPermissions Missing write access to /Users/sankris2_1/Library/Application Support/Lens/node_modules/lenscloud-lens-extension
npm WARN Lens No description
npm WARN Lens No repository field.
npm WARN Lens No license field.

npm ERR! code ENOENT
npm ERR! syscall access
npm ERR! path /Users/sankris2_1/Library/Application Support/Lens/node_modules/lenscloud-lens-extension
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, access '/Users/sankris2_1/Library/Application Support/Lens/node_modules/lenscloud-lens-extension'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sankris2_1/.npm/_logs/2022-06-09T16_20_42_614Z-debug.log

**Comment by Nokel81** [comment]
If you remove the folder `/Users/sankris2_1/Library/Application Support/Lens/node_modules` and the file `/Users/sankris2_1/Library/Application Support/Lens/lens-extensions.json` while Lens is quit and then start it up again does that solve the issue?

**Comment by santikris2003** [comment]
/Users/sankris2_1/Library/Application Support/Lens/lens-extensions.json was not present in the said path and I removed the folder -/Users/sankris2_1/Library/Application Support/Lens/node_modules and started the Lens 
It solved the problem can see now >-Shell option to choose as attached screenshot
<img width="176" alt="Screen Shot 2022-06-09 at 10 11 03 PM" src="https://user-images.githubusercontent.com/51011247/172899863-4c1ca49c-9431-4ad6-bc57-52b1c2b551b2.png">

**Comment by santikris2003** [comment]
appreciate your time to resolve this

**Comment by Nokel81** [comment]
Cool, will leave this open as we should handle that case better.
